x-pack/plugins/endpoint/server/routes/whitelist.ts

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
This file's pretty long, maybe think about splitting it up?

Reply:
Yeah definitely